py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -137,6 +137,24 @@ def aten_ops_sigmoid(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.index.Tensor)
```

Comment:
Consider adding `@enforce_tensor_types( {0: (TRTTensor,)} ), to ensure the input is a `TRTTensor`